grpc/tests/grpctest.cpp

```diff
@@ -93,24 +111,42 @@ int main(int /*argc*/, const char * /*argv*/[]) {
                                      grpc::InsecureChannelCredentials());
   auto stub = MyGame::Example::MonsterStorage::NewStub(channel);
 
-  grpc::ClientContext context;
-
-  // Build a request with the name set.
-  flatbuffers::FlatBufferBuilder fbb;
-  auto monster_offset = CreateMonster(fbb, 0, 0, 0, fbb.CreateString("Fred"));
-  fbb.Finish(monster_offset);
-  auto request = flatbuffers::BufferRef<Monster>(fbb.GetBufferPointer(),
-                                                 fbb.GetSize());
-  flatbuffers::BufferRef<Stat> response;
-
-  // The actual RPC.
-  auto status = stub->Store(&context, request, &response);
-
-  if (status.ok()) {
-    auto resp = response.GetRoot()->id();
-    std::cout << "RPC response: " << resp->str() << std::endl;
-  } else {
-    std::cout << "RPC failed" << std::endl;
+  {
+    grpc::ClientContext context;
+    // Build a request with the name set.
+    flatbuffers::FlatBufferBuilder fbb;
+    auto monster_offset = CreateMonster(fbb, 0, 0, 0, fbb.CreateString("Fred"));
+    fbb.Finish(monster_offset);
+    auto request = flatbuffers::BufferRef<Monster>(fbb.GetBufferPointer(),
+                                                   fbb.GetSize());
+    flatbuffers::BufferRef<Stat> response;
+
+    // The actual RPC.
+    auto status = stub->Store(&context, request, &response);
+
+    if (status.ok()) {
+      auto resp = response.GetRoot()->id();
+      std::cout << "RPC response: " << resp->str() << std::endl;
+    } else {
+      std::cout << "RPC failed" << std::endl;
+    }
+  }
+  {
+      grpc::ClientContext context;
```

Comment:
Ok, we can reuse FlatBufferBuilder, but ClientContext is not.
It happens asset like this
```
E0508 17:56:15.072436359     296 client_context.cc:97]       assertion failed: call_ == nullptr
```
from https://github.com/grpc/grpc/blob/master/src/cpp/client/client_context.cc#L98.